### PR TITLE
Beekeeping 3.7

### PR DESCRIPTION
**Pull request by DeityLink**
I mean, we never stopped to ask "why exactly are we putting apiaries in hydroponic trays?"

image

:cl:
* tweak: Apiaries no longer require you to build them into an hydroponics tray! Just click an adjacent floor while holding the kit.
* imageadd: Apiaries have new sprites to commemorate their graduation from being tray-bound.
* imageadd: Apiary kits now have in-hand icons.
* tweak: You no longer need a hatchet to harvest an apiary and can now do so bare-handed. Doing it this way no longer deconstructs the apiary so you can just calm the bees and let them return to it on their own. Hatchets (or alternatively crowbars) are still required to deconstruct them, which still also harvests them at the same time if they contain honey.
* soundadd: Added sounds for building/deconstructing apiaries, and for pouring BeezEz in them.
* rscadd: Apiaries and hives can burn down at long last.

### CODE REVIEW

**Comment by wotisjanitor** [comment]
secret hornet nerf

**Comment by DeityLink** [comment]
Being able to burn down hornet nests was always intended, it was just easier to finally implement with west's directions (pun-unintended).

File: code/game/machinery/bees_apiary.dm [hunk 1]
```diff
@@ -73,53 +76,78 @@ var/list/apiaries_list = list()
...
+			I = image('icons/obj/apiary_bees_etc.dmi', icon_state="[apiary_icon]_honey_2")
 		if(90 to INFINITY)
-			I = image('icons/obj/apiary_bees_etc.dmi', icon_state="honey_3")
+			I = image('icons/obj/apiary_bees_etc.dmi', icon_state="[apiary_icon]_honey_3")
 	if(!I)
 		return
 	I.color = mix_color_from_reagents(reagents.reagent_list)
 	overlays += I
+	if (on_fire)
+		overlays += fire_overlay
```
Comment (west3436) [inline code/game/machinery/bees_apiary.dm:119]: Did you test to ensure this doesn't result in stacked fire overlays?

**Review by west3436 — APPROVED** [review]

File: code/game/machinery/bees_apiary.dm (hunk 1 above)
Comment (DeityLink) [reply code/game/machinery/bees_apiary.dm:119]: The first line of the proc removes every overlays. So no but I did test burning hives before adding that and the fire overlay went away any time udpate_icon()  was called, so this actually fixes that.

**Pull request merged**